lib/metadata/ocsp/client.rb

```diff
@@ -16,13 +16,23 @@ def check(certificates, issuer, store)
           hash
         }
         ocsp_uri = find_ocsp_uri(certificates.first)
-        ocsp_request = build_ocsp_request_body(cert_ids.values)
-        ocsp_response = make_ocsp_request(ocsp_uri, ocsp_request)
-        cert_id_statuses = check_response(ocsp_request, ocsp_response, cert_ids.values, store)
-        certificates.inject({}) { |hash, certificate|
-          hash[certificate] = cert_id_statuses[cert_ids[certificate]]
-          hash
-        }
+        unless ocsp_uri.nil?
```

Comment:
stylistically, `unless foo.nil? ... else` has three negatives in it which makes it hard to understand.  I think I'd prefer at least doing `if foo.nil? .. else` and swapping the bodies round but there may be other ways to express this.